src/components/Modal/EditThemeModal/EditThemeModal.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <modal-frame title="カスタムテーマ" icon-name="edittheme">
```

Comment:
icon-nameに存在しないアイコン名指定しててエラー出てるのですが、コンポーネント自体を直す必要があって僕が直してる途中なので、一旦空文字にしておいてほしいです:pray:
```suggestion
  <modal-frame title="カスタムテーマ" icon-name="">
```
![image](https://github.com/traPtitech/traQ_S-UI/assets/83744975/bd0c4add-054e-43d5-ae05-3beb3e51886a)